Services/UserService/Interfaces/IDeleteUser.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Net;
+
+namespace IDeleteUserUserService.Interfaces
+{
+
+    public interface IDeleteUser
+    {
+        HttpStatusCode Execute(int userId);
```

Comment:
No need to return HttpStatusCode, **void** is good here and if you want something more you can use **ActionResult**